adafruit_bitbangio.py

```diff
@@ -23,6 +23,14 @@
 
 """
 
+try:
+    from typing import Optional
+    from typing_extensions import Literal
+    from circuitpython_typing import WriteableBuffer, ReadableBuffer
+    from microcontroller import Pin
```

Comment:
Based on this import, make sure you're using `Pin` instead of `microcontroller.Pin` in your type annotations!